config.yaml

```diff
@@ -0,0 +1,5 @@
+exclude_extensions: [ ".log", ".jpg", ".jpeg", ".png", ".gif", ".bmp", ".tiff", ".tif", ".psd", ".xcf", ".zip", ".tar.gz",".gz",".so", ".0", ".1", ".2", ".3",".4",".5",".6",".7",".8",".9", ".ttf", ".lock", ".yar", ".log", ".chk", ".sdb", ".jdb", ".pat", ".jrs", ".dit", ".pol", ".mdb", ".dns", ".admx", ".adml", ".adm", ".edb", ".db", ".evtx"]
+# need to confirm as windows hides file extensions
+exclude_paths: ["{sep}var{sep}lib{sep}docker", "{sep}var{sep}lib{sep}containerd", "{sep}dev", "{sep}proc", "{sep}usr{sep}lib", "{sep}sys", "{sep}boot", "{sep}run"] # use {sep} for the OS' path seperator (i.e. / or \)
```

Comment:
Do we really care about that `{sep}` thing?
We are mostly running on linux, we should simplify this by simply using `/`